docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: "3.9"
+   
+services:
+  db:
+    image: mysql
+    ports:
+      - '3306:3306'
+    environment:
+      MYSQL_DATABASE: 'zues'
+      MYSQL_USER: 'jd'
+      MYSQL_PASSWORD: 'jd2021!'
+      MYSQL_ROOT_PASSWORD: 'jd2021'
```

Comment:
Willen we de wachtwoorden nog populaten uit een local file? Anders misschien even noemen dat dit alleen in development gebruikt moet worden?